src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -1001,6 +1006,34 @@ def __repr__(self):
         return repr_str
 
 
+class ConvertBox(tvt_v2.Transform):
+    """Convert bounding boxes to a different format."""
+
+    _transformed_types = (tv_tensors.BoundingBoxes,)
+
+    def __init__(self, out_fmt: str = "", normalize: bool = False) -> None:
```

Comment:
```suggestion
    def __init__(self, out_fmt: Literal["xyxy", "cxcywh"] = "xyxy", normalize: bool = False) -> None:
```
It seems that `out_fmt` should be one of these values because If out_fmt == "", `inpt` isn't initialized in `_transform` function, which makes an error. If it's right, how about updating type checking like this and removing `if self.out_fmt:` in `_transform` function?

Reply:
It seems for a condition in L1025, doesn't it?

Reply:
you're right. I deluded myself. Thanks for pointing it.